analyses/transcriptomic-dimension-reduction/04-explore-sequencing-center-effects.Rmd

```diff
@@ -505,14 +512,12 @@ cg_plot +
     "segment",
     x = -2, y = 0.25, xend = -1, yend = 0.25,
     color = "red", lwd=1.5,
-    arrow = arrow(), 
-    show.legend = FALSE) + 
+    arrow = arrow()) + 
```

Comment:
Just wondering about this change. Is `show.legend=FALSE` not needed?

Reply:
Yep, turns out it's an argument that doesn't do anything but toss a warning at you that it's not a real argument in the `ggplot2` version we have - 

> While I was here I also removed a ggplot argument `show.legend = FALSE` which apparently is not in our ggplot version, and it throws a warning during runtime and then does absolutely nothing in the plot. So, it's gone!